models/interval.go

```diff
@@ -37,20 +37,14 @@ func (i Interval) MarshalJSON() ([]byte, error) {
 	test := struct {
 		Timestamps
 		ID        *string `json:"id,omitempty"`
-		Created   int64   `json:"created,omitempty"`
-		Modified  int64   `json:"modified,omitempty"`
-		Origin    int64   `json:"origin,omitempty"`    // not currently used reserved for interval origin source
 		Name      *string `json:"name,omitempty"`      // non-database identifier for a shcedule (*must be quitue)
 		Start     *string `json:"start,omitempty"`     // Start time i ISO 8601 format YYYYMMDD'T'HHmmss 	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyymmdd'T'HHmmss")
 		End       *string `json:"end,omitempty"`       // Start time i ISO 8601 format YYYYMMDD'T'HHmmss 	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyymmdd'T'HHmmss")
 		Frequency *string `json:"frequency,omitempty"` // how frequently should the event occur
 		Cron      *string `json:"cron,omitempty"`      // cron styled regular expression indicating how often the action under schedule should occur.  Use either runOnce, frequency or cron and not all.
 		RunOnce   bool    `json:"runOnce,omitempty"`   // boolean indicating that this interval runs one time - at the time indicated by the start
 	}{
-		Created:  i.Created,
-		Modified: i.Modified,
-		Origin:   i.Origin,
-		RunOnce:  i.RunOnce,
+		RunOnce: i.RunOnce,
```

Comment:
There's a discrepancy here. The Timestamps type is a struct that contains Created, Modified and Origin properties. These have been removed as separate properties from the marshaling struct, but not from the Interval type itself. Also, you would need to set the Timestamps struct so the values are retained
```
test := struct {
    Timestamps
    ...
}{
    Timestamps: i.Timestamps
    ...
}
```